libr/core/cmd_anal.c

```diff
@@ -5740,6 +5740,53 @@ static void agraph_print_node(RANode *n, void *user) {
 	free (encbody);
 }
 
+static char *getViewerPath() {
+	int i;
+	const char *viewers[] = {
+		"open",
```

Comment:
Where are my changes? :(

Reply:
Wait, maybe I fucked up the merge of the conflicts, let me check

Reply:
I copied the old getViewerPath and left the new one in cconfig.c :(
Sorry, my bad, will fix it